api/Window.json

```diff
@@ -4197,10 +4197,10 @@
               "version_added": null
             },
             "safari": {
-              "version_added": null
+              "version_added": true
             },
             "safari_ios": {
-              "version_added": null
+              "version_added": true
```

Comment:
Out of curiosity, where did you get the data from?

Reply:
I just tried it on current safari and it worked. Plugged in phone and used developers tools. Was able to run some of the functions but not others